src/routes/tools/slr-coastal-inundation/_Map/Rasters.svelte

```diff
@@ -39,12 +39,14 @@
     .filter((d) => Array.isArray(d.tileUrls) && d.tileUrls.length)
     .map(mapLayersProps);
 
-  // TODO: figure out why 5m layer isn't added when switching from z6 to z7
-  $: if (!equal(rasterLayersProps, prevRasterLayerProps)) {
-    removePreviousRasterLayers();
-    addRasterLayers();
-    prevRasterLayerProps = rasterLayersProps;
-  }
+  beforeUpdate(async () => {
+    await tick();
```

Comment:
@mukhtyar this fixed a bug for me with the map layers getting out of sync, we should probably use `tick` elsewhere for similar situations that rely on async processes.

Reply:
Actually there's a better example in the CAE website repo I can show you, I ended up using `afterUpdate` here instead of `beforeUpdate` with `tick`